docs/upgrade/focal_migration.rst

```diff
@@ -0,0 +1,240 @@
+Ubuntu 20.04 LTS (Focal) Migration Guide
+========================================
+
+On April 30, 2021, Ubuntu 16.04 LTS (Xenial), the operating system for the
+SecureDrop servers, will reach End of Life. In order to continue using SecureDrop, 
+instances must migrate to Ubuntu 20.04 LTS (Focal) **before** April 30, 2021.
+
+This migration will require on-premises access to the servers, and a complete
+reinstallation of Ubuntu and SecureDrop. In-place upgrades and 
+remote upgrades via SSH are not currently supported.
+
+.. important::
+
+   For security reasons, the *Source Interface* will automatically be
+   disabled on SecureDrop servers still running Ubuntu 16.04 after 
+   April 30, 2021.
+
+We recommend that you plan a two day maintenance window
+**between March 9 and April 30** to perform and test the migration.
```

Comment:
Since it's not currently noted in this doc, and we might link directly to subsections of the backup/restore guide, I do think it may be worth emphasizing here or in a similar location that data on the *Monitor Server* is not preserved as part of this migration.

Reply:
I have mentioned it as a callout in each of the migration paths (d86780e). I did this in order to keep the info at the top as concise and important as possible. Feedback welcome!

Reply:
I think the way you have introduced this warning now exactly when it is most relevant is perfect, thanks. :)